pkg/build/builder/docker.go

```diff
@@ -87,6 +87,31 @@ func (d *DockerBuilder) Build() error {
 	}
 
 	buildTag := randomBuildTag(d.build.Namespace, d.build.Name)
+	dockerfilePath := d.getDockerfilePath(buildDir)
+	imageNames := getDockerfileFrom(dockerfilePath)
+	if len(imageNames) == 0 {
+		return fmt.Errorf("no from image in dockerfile.")
+	}
+	for _, imageName := range imageNames {
+		var imageExists bool = true
+		_, err = d.dockerClient.InspectImage(imageName)
+		if err != nil && err == docker.ErrNoSuchImage {
+			imageExists = false
+		} else if err != nil {
+			return err
+		}
+		// if forcePull or the image not exists on the node we should pull the image first
+		if d.build.Spec.Strategy.DockerStrategy.ForcePull || !imageExists {
+			pullAuthConfig, _ := dockercfg.NewHelper().GetDockerAuth(
+				imageName,
+				dockercfg.PullAuthType,
+			)
+			glog.V(0).Infof("\nPulling image %s ...", imageName)
```

Comment:
Is pulling an image truly something that should always be shown?  Did we show it before?